app/assets/javascripts/admin/addon/templates/admin-badges/show.hbs

```diff
@@ -284,6 +289,20 @@
       >
         <field.Checkbox />
       </group.Field>
+
+      <group.Field
```

Comment:
`this.siteSettings.show_badges_in_post_header` is `false` and I'm wondering if we should just hide this field completely when it is so?

Reply:
That'd make this option very difficult to discover organically, you'd have to know how to get to it first before you could enable it.

Reply:
This is fair, thank you for clarifying 👍